src/Features/SkySync.h

```diff
@@ -0,0 +1,175 @@
+﻿#pragma once
+#include "RE/M/Moon.h"
+
+struct SkySync : Feature
+{
+	static SkySync* GetSingleton()
+	{
+		static SkySync singleton;
+		return &singleton;
+	}
+
+	virtual inline std::string GetName() override { return "Sky Sync"; }
+	virtual inline std::string GetShortName() override { return "SkySync"; }
+
+	struct Settings
+	{
+		bool Enabled = true;
+		bool UseAlternateSunPath = true;
+		int32_t MoonLightSource = 0;
+	};
+
+	Settings settings;
+
+	virtual void DrawSettings() override;
+
+	virtual void LoadSettings(json& o_json) override;
+	virtual void SaveSettings(json& o_json) override;
+	virtual void RestoreDefaultSettings() override;
+
+	virtual bool SupportsVR() override { return true; }
```

Comment:
Let's also make sure VR Address Library is set to 0.175.0. We've missed that in the last few prs. https://github.com/doodlum/skyrim-community-shaders/blob/c188df38a9ff0ac03344e827182ce1840b0256ca/src/XSEPlugin.cpp#L156

Reply:
Ah cheers didn't know this was a thing, updated in latest.